R/net.accessor.R

```diff
@@ -260,7 +259,7 @@ get_epi_list <- function(dat, item = NULL) {
 
 #' @rdname net-accessor
 #' @export
-get_epi <- function(dat, item, indexes = NULL, override.null.error = FALSE) {
+get_epi <- function(dat, item, posit_ids = NULL, override.null.error = FALSE) {
```

Comment:
posit_ids does not have the same meaning for epi as it does for attr, because the time step in epi is not really an ID conceptually but just a marker for the time step, at. Can we just use `at` for the argument name?